src/sentry/static/sentry/app/utils/withMeasurements.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {MeasurementCollection} from 'app/types';
+
+type InjectedMeasurementsProps = {
+  measurements: MeasurementCollection;
+};
+
+const MEASUREMENTS: MeasurementCollection = {
+  'measurements.fp': {name: 'First Paint', key: 'measurements.fp'},
+  'measurements.fcp': {name: 'First Contentful Paint', key: 'measurements.fcp'},
+  'measurements.lcp': {name: 'Largest Contentful Paint', key: 'measurements.lcp'},
+  'measurements.fid': {name: 'First Input Delay', key: 'measurements.fid'},
+};
+
+const withMeasurements = <P extends InjectedMeasurementsProps>(
```

Comment:
Are you thinking that this will expand in the future to fetch custom measurements?

Reply:
That was the intent.